component-samples/demo/owner/README.md

```diff
@@ -92,22 +91,20 @@ Make sure to export the credential environment variables set in `service.env` fi
 
 ##  Run as Docker Service
 
-Open a terminal, change directory to `<fdo-pri-src>/component-samples/owner/` and execute following command.
+Open a terminal, change directory to `<fdo-pri-src>/component-samples/demo/owner/` and execute following command.
 
 ```
 docker-compose up --build
 ```
 
 In case you need super user access, prefix 'sudo -E' to above command.
 
-***NOTE :*** To support OnDie ECDSA Device attestation, copy the required certificates and crls to `<fdo-pri-src>/component-samples/owner/ondiecache` folder.
+***NOTE :*** To support OnDie ECDSA Device attestation, copy the required certificates and crls to `<fdo-pri-src>/component-samples/demo/owner/ondiecache` folder.
```

Comment:
Since in this implementation, Ondie artifacts are present in csme.zip/cmse-debug.zip and located in /demo/aio path, this line can be removed and a redirect link to Configuring Ondie section present in Main README can be included here.

Reply:
To configure Ondie, refer https://github.com/secure-device-onboard/pri-fidoiot/tree/master/component-samples/demo#configuring-ondie-optional